rust/utils/run-locally/src/main.rs

```diff
@@ -29,16 +34,27 @@ use nix::{
 };
 use tempfile::tempdir;
 
+const RELAYER_KEYS: &[&str] = &[
+    "8166f546bab6da521a8369cab06c5d2b9e46670292d85c875ee9ec20e84ffb61",
+    "f214f2b2cd398c806f84e317254e0f0b801d0643303237d97a22a48e01628897",
+    "701b615bbdfb9de65240bc28bd21bbc0d996645a3dd57e7b12bc2bdf6f192c82",
+];
+const VALIDATOR_KEYS: &[&str] = &[
```

Comment:
nit: Please add a comment saying these must be consistent with the ism config for test (and please add a corresponding comment to that config)